webpack.index-worker.config.js

```diff
@@ -44,14 +42,6 @@ const config = {
 
 if (!process.env.development) {
   config.mode = "production";
-  config.plugins = [...(config.plugins || []),
```

Comment:
Is there anything left preventing an easy transition of undoing the ejection?

Reply:
?

Reply:
I have no idea honestly, but we're using Craco I think, so we may have not

Reply:
Maybe we can eject, but it's not in this PR's scope, if we ever want to do that I'd rather do it some other time.

Reply:
@Urook asked about undoing eject, but I don't think we ever did it. I think we can resolve this